features/bootstrap/support.php

```diff
@@ -1,38 +1,42 @@
-<?php
+<?php namespace WP_CLI\Support;
```

Comment:
May I suggest for the `namespace` declaration to be on its own line ?

While there is currently no sniff in place to check this, I expect one to be introduced soon enough.
```suggestion
<?php
 namespace WP_CLI\Support;
```

Reply:
Oh... and just thinking - should the namespace possibly be `WP_CLI\Tests\Support` ?

Reply:
As the namespace has been adjusted now, are there any `use` statements which need adjusting too ?
I seem to remember seeing one in `steps/then.php`... ?